src/ResourceManager/DevSpaces/Commands.DevSpaces/help/Get-AzureRmDevSpacesController.md

```diff
@@ -0,0 +1,165 @@
+ [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Get-AzureRmDevSpacesController -InputObject <PSController> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get or list Azure DevSpaces controller.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmDevSpacesController
+```
+
+List all controllers in a subscription.
+
+### Example 2
+```powershell
+PS C:\> Get-AzureRmDevSpacesController --ResourceGroupName devSpaceResourceGroup -Name devSpaceControllerName
+```
+
+Get a DevSpaces controllers in a subscription.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+A PSController object, normally passed through the pipeline.
+
+```yaml
+Type: PSController
+Parameter Sets: InputObjectParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Name
+DevSpaces controller name.
+
+```yaml
+Type: String
+Parameter Sets: DevSpacesControllerNameParameterSet
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Resource group name
+
+```yaml
+Type: String
+Parameter Sets: ListDevSpacesControllerParameterSet
+Aliases:
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+```yaml
+Type: String
+Parameter Sets: DevSpacesControllerNameParameterSet
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceId
+The DevSpaces resource id
+
+```yaml
+Type: String
+Parameter Sets: ResourceIdParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### System.String
+Microsoft.Azure.Commands.DevSpaces.Models.PSController
+
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.DevSpaces.Models.PSController
+System.Collections.Generic.IList`1[[Microsoft.Azure.Commands.DevSpaces.Models.PSController, Microsoft.Azure.Commands.DevSpaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
Remove this second one - we will be adding links to outputs soon, so just want the type, not the list part.

Reply:
Done

Reply:
Ping

Reply:
Sorry, it got generated again.